homeassistant/components/squeezebox/media_player.py

```diff
@@ -397,55 +364,68 @@ def query_result(self) -> dict | bool:
     async def async_turn_off(self) -> None:
         """Turn off media player."""
         await self._player.async_set_power(False)
+        await self.coordinator.async_refresh()
 
     async def async_volume_up(self) -> None:
         """Volume up media player."""
         await self._player.async_set_volume("+5")
+        await self.coordinator.async_refresh()
 
     async def async_volume_down(self) -> None:
         """Volume down media player."""
         await self._player.async_set_volume("-5")
+        await self.coordinator.async_refresh()
 
     async def async_set_volume_level(self, volume: float) -> None:
         """Set volume level, range 0..1."""
         volume_percent = str(int(volume * 100))
         await self._player.async_set_volume(volume_percent)
+        await self.coordinator.async_refresh()
```

Comment:
Not sure if you tested it, but if you use `async_request_refresh` it will wait a slight moment before fetching new data so it can group the entities. But I can understand that this is a local service so we don't really care that much if we do multiple requests. Just FYI that this is a possibility, I think you can wager if that's needed or not

Reply:
Thanks. I went with async_refresh because it is local and because the UI feels nicer with instant feedback